src/App.tsx

```diff
@@ -1,18 +1,29 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
+import React from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
 
 import './App.scss';
 
-export const App = () => {
+import { Navbar } from './components/Navbar';
+import { PeoplePage } from './pages/PeoplePage';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const App: React.FC = () => {
   return (
     <div data-cy="app">
       <Navbar />
 
       <div className="section">
         <div className="container">
```

Comment:
if I decided to add a new page which will not include a `Navbar `, how can I do this?
I suggest you create a `Layout` component and wrap all your routes in the `Layout` route.
```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside.